add k2 content strategy draft (#565)
kyma-project/community: guidelines/content-guidelines/01-content-strategy-k2.md

>
> 1. A condition or symptom. Usually the condition or symptom is an undesirable state in a system, a product, or a service that a reader wants to correct.
> 2. A cause for the condition or symptom.
> 3. A remedy for the condition or symptom.
>
> The troubleshooting topic provides sections for describing the condition, causes, and remedies needed to restore a system, a product, or a service to normal.
>
> For some conditions there could be more than one cause-remedy pair. The troubleshooting topic accommodates this. Typically, a cause is immediately followed by its remedy. Multiple cause-remedy pairs can provide a series of successive fall-backs for resolving a condition.
>
> Cause and remedy might occur in combinations other than pairs. It is possible to have:
>
> * Multiple causes with the same remedy
> * A single cause with more than one remedy
> * A remedy with no known cause
> * A cause with no known remedy
> The troubleshooting information type also can be used to document alarm clearing strategies.
> [- DITA Troubleshooting Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-troubleshooting-topic.html)

> ## Graphics

Reviewer: We have separate document for [diagrams](https://github.com/kyma-project/community/blob/main/guidelines/content-guidelines/02-diagrams.md) and [screenshots](https://github.com/kyma-project/community/blob/main/guidelines/content-guidelines/07-screenshots.md), but of course we can link them here.

Author: Not sure if the section will stay or if we remove it entirely because the other documents cover that.

Reviewer: We can leave this section and only provide links to the diagram and screenshot guidelines.

### Architecture and Flow Diagrams

*Klaudia is preparing the new diagram style, see <https://github.com/kyma-project/community/issues/542>.*

### Screenshots

Wherever possible, present screenshots as SUI (simplified user interfaces). Basically, this means to blur out all UI elements that aren't essential for the task at hand.

For more information, see:
• <https://www.techsmith.com/blog/simplified-user-interface/>
• <https://www.tcworld.info/e-magazine/technical-writing/simplified-graphics-and-screenshots-in-software-documentation-1102/>

> ## Documentation Structure and Target Groups

Reviewer: I would combine the documentation structure and target groups with information types that you described above, for example by listing our doc types (overview, getting started, etc.) and adding information about who is the target groups for this doc type and what information type is expected there. I really like the excel sheet you prepared some time ago, but I also know that tables in Markdown are quite inconvenient to use 😕 Let's discuss.

Author: I believe information type and target group don't always need to coincide (even though there might be overlaps - more concept topics in the Overview section). 
There might be conceptual information (for example, about profiles) that's too detailed to newbies/decision makers, but is interesting for admins.

Author: I could convert the information to a table once we're fairly sure about the content. But I believe editing is going to be a hassle, so we could keep the normal paragraphs while it's work in progress.

On the Kyma website, we’ll have five main tabs containing multiple documents each, plus a glossary.

### Overview

Target Group: Decision Makers (Tech Leads) and newbies.
Contains a quick overview of the idea behind Kyma, a diagram of the main areas with a brief explanation, and, as needed, longer documents going into the details of each main area.

### Getting Started

Target Group: Software Developers who quickly want to see what they can do with Kyma.

> Contains a guide/tutorial designed by PB.

Reviewer: 😆

Author: I'll do my best to remember updating this before we go live!

Author: Updated! :D

### Deep Dive/Tutorials

Target Group: Software Developers leveraging all the Kyma functionalities.
Under this tab, there will be subtabs according to main areas (except UI – Busola and CLI are mentioned as needed within the instructions of the respective main area.)

### Operations

Target Group: Admins/Operators who make sure the Kyma cluster is configured as needed and keeps running in a healthy and secure way.
Contains installation and configuration instructions, backup info, troubleshooting guides…

### Technical References

Target Group: Users who want need to look up specific detailed information.
Contains the architecture diagrams, configuration charts, etc,; no explanation of concepts or instructions.

### Glossary

Target Group: Anyone who wants to look up terms they’re not familiar with.
Explains basic terms, with a focus on terms specific to Kyma.

## Content Source

> We write the content in [Markdown](https://daringfireball.net/projects/markdown/) and store it in [Git](https://git-scm.com/) repositories.

Reviewer: I'd also add `The assumed reader's knowledge` and `Release notes` sections that were also included in the old content strategy.

Author: Good point!! Thank you.

Should we include `Release Notes` as an information type? It's got a defined structure that's doesn't really the existing info types.

Reviewer 2: I think Klaudia meant [this](https://github.com/kyma-project/community/blob/main/guidelines/content-guidelines/01-content-strategy.md#the-assumed-readers-knowledge) section.

Author: I added that!

- [Target Groups](#target-groups)
  - [Decision maker](#decision-maker)
  - [Software developer](#software-developer)
  - [Admin/Operations](#adminoperations)
- [Documentation Structure](#documentation-structure)
  - [Overview](#overview)
  - [Getting Started](#getting-started)
  - [Deep Dive/Tutorials](#deep-divetutorials)
  - [Operations](#operations)
  - [Technical References](#technical-references)
  - [Glossary](#glossary)
- [Content Source](#content-source)

## Purpose and Audience

This content strategy focuses on the publicly available documentation under <https://kyma-project.io/docs/>.
More documentation may be found in the Kyma Github repositories.

The assumed reader of this guide has some basic knowledge of technical writing. To learn more, read the excellent [Istio document about adding documentation](https://istio.io/latest/docs/releases/contribute/add-content/).

## Information types

We follow a topic-based documentation approach, with one file per topic. Every documentation file has a clearly defined purpose, which is reflected in the title. The content must be able to stand on its own, but links should point to other documents as needed. 

> *For the structure within topics, I suggest to follow the DITA standard – because it’s pretty simple, it’s well-established, and it’s what is used in SKR documentation. Instead of currently 12-13 content types, we should thus be able to get away with just four (or five) types:*

Author: I just put it there to start the discussion - once we're all aligned on the preferred approach, I'll remove it. So far, I'm not yet sure whether this suggestion is fine with all.

### Concept topics

Answer "what-is" questions and provide essential background information that users must know.
You'll find most concept topics in the Overview section, but they can be useful elsewhere too.

Use nominal style for the title, for example, "Security" or "Security Concept".

> *Not sure we need much predefined structure here; it may depend on the content.*

Author: I hope someone chimes in with an opinion!

> > [- DITA Concept Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-concept-topic.html)

Author: I just regard them as block quotes - we don't even need to keep them in the final version. I just wanted to add them so that we have a baseline to discuss what we want.

> > [- DITA Concept Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-concept-topic.html)

Author: removed them anyway :D

> Conceptual information might explain the nature and components of a product and describe how it fits into a category of products. Conceptual information helps readers to map their knowledge and understanding to the tasks they need to perform and to provide other essential information about a product, process, or system.

> You can use the [concept topic template](/templates/resources/concept.md).

Author: thank you! I'm still a noob with markdown links

### Task (tutorial) topics

Provide "how-to" instructions that enable users to accomplish a task. Each task topic should tell how to perform a single, specific procedure.

Select a title that describes the task that's accomplished, not the documented software feature - for example, "Define ressource consumption", not "Select a profile". You can use gerund form "Selecting...", imperative "Select...", or "How to select...".

With regards to structure, it’s nice to have an **introductory paragraph** ("why would I want to do this task?"), **prerequisites** if needed, then the **steps**, and finally the expected **result** that shows the operation was successful.
It's good practice to have 5-9 steps; anything longer can probably be split.

> *For longer code blocks, I'd like to use an expandable section as described [here](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab) - but not sure whether our website supports this.*

Author: I love our toggle, but it doesn't really help with collapsing/expanding code blocks. I mean behaviour like in the following examples:
- https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab 
- https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/9b178ab3388c4647b0c52f2c85641844.html

> [- Dita Task Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-task-topic.html)
> A task information type answers the "How do I?" question by providing precise step-by-step instructions detailing the requirements that must be fulfilled, the actions that must be performed, and the order in which the actions must be performed. The task topic includes sections for describing the context, prerequisites, expected results, and other aspects of a task.

You can use the [task topic template](/templates/resources/task.md).

### Reference topics

Typically organized into one or more sections containing a list or table with data that is typically “looked up” rather than memorized.

Use nominal style for the title, for example, "Configuration Parameters".

> *In our case, architecture diagrams could fall into this category – we could also choose to define them as a separate type – TBD.*

Author: keep until we found a solution that we like, then remove :)

> *In our case, architecture diagrams could fall into this category – we could also choose to define them as a separate type – TBD.*

Author: Should we create separate templates for them?

>
> Cause and remedy might occur in combinations other than pairs. It is possible to have:
>
> - Multiple causes with the same remedy
> - A single cause with more than one remedy
> - A remedy with no known cause
> - A cause with no known remedy
> The troubleshooting information type also can be used to document alarm clearing strategies.

You can use the [troubleshooting topic template](/templates/resources/troubleshooting.md).

### Release Notes

Announce what's new in Kyma. 

After an introductory paragraph that outlines the city that's the namesake of the current release, a list briefly presents the new and changed features. Links lead to longer paragraphs that describe the changes in more detail.

## Graphics

### Architecture and Flow Diagrams

*Klaudia is preparing the new diagram style, see <https://github.com/kyma-project/community/issues/542>.*

### Screenshots

Wherever possible, present screenshots as SUI (simplified user interfaces). Basically, this means to blur out all UI elements that aren't essential for the task at hand.

> For more information, see:

Author: arrrggghl. Thank you for noticing. I used the wrong format.

• <https://www.techsmith.com/blog/simplified-user-interface/>
• <https://www.tcworld.info/e-magazine/technical-writing/simplified-graphics-and-screenshots-in-software-documentation-1102/>

## Target Groups

> 

Author: Which kind of introductory sentence would you suggest?

---

## Table of contents

- [Purpose and Audience](#purpose-and-audience)
- [Information types](#information-types)
  - [Concept Topics](#concept-topics)
  - [Task (tutorial) topics](#task-tutorial-topics)
  - [Reference topics](#reference-topics)
  - [Troubleshooting topics](#troubleshooting-topics)
  - [Release Notes](#release-notes)
- [Graphics](#graphics)
  - [Architecture and Flow Diagrams](#architecture-and-flow-diagrams)
  - [Screenshots](#screenshots)
- [Target Groups](#target-groups)
  - [Decision maker](#decision-maker)
  - [Software developer](#software-developer)
  - [Admin/Operations](#adminoperations)
- [Documentation Structure](#documentation-structure)
  - [Overview](#overview)
  - [Getting Started](#getting-started)
  - [Deep Dive/Tutorials](#deep-divetutorials)
  - [Operations](#operations)
  - [Technical References](#technical-references)
  - [Glossary](#glossary)
- [Content Source](#content-source)

## Purpose and Audience

This content strategy focuses on the publicly available documentation under https://kyma-project.io/docs/.
More documentation may be found in the [Kyma](https://github.com/kyma-project/kyma) GitHub repositories.

> The assumed reader of this guide has some basic knowledge of technical writing. To learn more, read the excellent [Istio document about adding documentation](https://istio.io/latest/docs/releases/contribute/add-content/).

Author: I'm not sure if our devs ever read our guides...? And I'd hope they also have some basic knowledge, otherwise we need to explain all the basic details - write concisely, don't use "please", avoid future tense, recommended sentence structure... :/ 

I agree about the Istio link though, dropping the link.

## Information types

> We follow a topic-based documentation approach, with one file per topic. Every documentation file has a clearly defined purpose, which is reflected in the title. The content must be able to stand on its own, but links should point to other documents as needed.

Author: nitpicking is a core quality in our job, bring it on! (I truly appreciate nitpicky review)

*For the structure within topics, I suggest to follow the DITA standard – because it’s pretty simple, it’s well-established, and it’s what is used in SKR documentation. Instead of currently 12-13 content types, we should thus be able to get away with just four (or five) types:*

### Concept topics

Answer "what-is" questions and provide essential background information that users must know.
You'll find most concept topics in the Overview section, but they can be useful elsewhere too.

> Use nominal style for the title, for example, "Security" or "Security Concept".

Author: for me "Allowed domains in the API Gateway Controller" is nominal style.

*Not sure we need much predefined structure here; it may depend on the content.*

> > [- DITA Concept Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-concept-topic.html)

Author: [DITA](https://en.wikipedia.org/wiki/Darwin_Information_Typing_Architecture) is an open standard so I don't think we can't mention it in OS docs. I wouldn't describe any CMS-specific procedures here, but the generic information types are pretty universal and I think we can use the ideas in markdown too, not just in xml.

> > [- DITA Concept Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-concept-topic.html)

Author: Alright! For now, I'm adding links at the end of the sections. The overview page for DITA topic types contains more than the ones we use and I'd like to point the reader to the correct one.

> Conceptual information might explain the nature and components of a product and describe how it fits into a category of products. Conceptual information helps readers to map their knowledge and understanding to the tasks they need to perform and to provide other essential information about a product, process, or system.

> You can use the [concept topic template](../templates/resources/concept.md).

Author: I personally don't care if the template was used if the result resembles the guidelines :D
I don't expect any of the Kyma TWs will use the template for every single new doc.
For anyone else creating a new doc templates might be helpful, though.

> ### Task (tutorial) topics

Author: Then I'd say we use "Task", as I would like to avoid that people mix up the navigation headline with the info type.

Provide "how-to" instructions that enable users to accomplish a task. Each task topic should tell how to perform a single, specific procedure.

> Select a title that describes the task that's accomplished, not the documented software feature. For example, use "Define resource consumption", not "Select a profile". You can use the gerund form "Selecting...", imperative "Select...", or "How to select...".

Author: Consistency would be great! I'm fine with the phrasing the majority prefers.

With regards to structure, it’s nice to have an **introductory paragraph** ("why would I want to do this task?"), **prerequisites** if needed, then the **steps**, and finally the expected **result** that shows the operation was successful.
It's good practice to have 5-9 steps; anything longer can probably be split.

*For longer code blocks, I'd like to use an expandable section as described [here](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab) - but not sure whether our website supports this.*

> [- Dita Task Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-task-topic.html)
> A task information type answers the "How do I?" question by providing precise step-by-step instructions detailing the requirements that must be fulfilled, the actions that must be performed, and the order in which the actions must be performed. The task topic includes sections for describing the context, prerequisites, expected results, and other aspects of a task.

> You can use the [task topic template](../templates/resources/task.md).

Author: I'm fine either way, up to the team.

### Reference topics

Typically organized into one or more sections containing a list or table with data that is usually “looked up” rather than memorized.

> Use nominal style for the title, for example, "Configuration Parameters".

Author: When I read "Switching registries at runtime", I expect to find an instruction how to switch registries at runtime, so, a task topic with a number of steps. Looking at the content of this one, I believe it implicitly does instruct users how to enable switching registries. So it might be better suited under "Operations" maybe?

*In our case, architecture diagrams could fall into this category – we could also choose to define them as a separate type – TBD.*

> > [- DITA Reference Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-reference-topic.html)

Reviewer 3: See my comment above re DITA.

Author: open standard.

> Reference topics provide quick access to fact-based information. In technical information, reference topics are used to list product specifications and parameters, provide essential data, and provide detailed information on subjects such as the commands in a programming language.

### Troubleshooting topics

> Provide a condition that the reader may want to correct, followed by one or more descriptions of its cause and suggested remedies.

Reviewer 3: ```suggestion
Provide a condition that the reader may want to correct, followed by descriptions of one or more of its cause and suggested remedies.
```

Author: I think this has a slightly different meaning?

> As title, mention the symptom that needs fixing ("Cannot access...") or the error message. To quote an error message, start and end with `'` to escape `"`. for example, `title: '"FAILED" status for created ServiceInstances'`. Do not use the cause as title ("Incompatible version"), because we also want to help users who have no idea about the cause and only know something's not working as expected.

Reviewer 3: I'm wondering if we're not going to shoot ourselves in the knee with providing the error in the title. 
On one hand, if we do this, it's great for the user 'cause they can find it easily when they encounter the problem. 
On the other hand:
- the user has the search tool and can find it easily anyway
- it's problematic and not very elegant when it comes to title formatting/rendering
- one cause can result in multiple errors, one error can have multiple causes, so these docs might grow if you want to describe it all in one doc.

I'd personally stick to using the cause as a title.

Author: I agree there is the search tool, but a good search tool shouldn't replace meaningful titles. 

Some users might prefer the search, others will rather scan the titles in the navigation. And as a user, I wouldn't know which cause to look out for (if I knew it, I'd probably be competent to fix it without our doc). So I'd prefer to use the problem for the title.

How about we ask others - the TWs, as well as potential users? What do our devs think, or the SREs? I'm fine with what the majority prefers.

It's good practice to use three standard headlines (like “Condition”, “Cause”, “Remedy”), each might have just one sentence or more as needed. For remedy, use a numbered list if there are multiple steps to follow, and a bullet list if there are several equally valid solutions.

> > [- DITA Troubleshooting Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-troubleshooting-topic.html)

Reviewer 3: See my comment above re DITA.

Reviewer: I agree with @majakurcius on this.

> > In its simplest form, troubleshooting information follows this pattern:

Reviewer: I'd add this info to the first paragraph.

>
> 1. A condition or symptom. Usually the condition or symptom is an undesirable state in a system, a product, or a service that a reader wants to correct.
> 2. A cause for the condition or symptom.
> 3. A remedy for the condition or symptom.
>
> The troubleshooting topic provides sections for describing the condition, causes, and remedies needed to restore a system, a product, or a service to normal.
>
> For some conditions there could be more than one cause-remedy pair. The troubleshooting topic accommodates this. Typically, a cause is immediately followed by its remedy. Multiple cause-remedy pairs can provide a series of successive fall-backs for resolving a condition.
>
> Cause and remedy might occur in combinations other than pairs. It is possible to have:
>
> - Multiple causes with the same remedy
> - A single cause with more than one remedy
> - A remedy with no known cause
> - A cause with no known remedy
> The troubleshooting information type also can be used to document alarm clearing strategies.

> You can use the [troubleshooting topic template](../templates/resources/troubleshooting.md).

Reviewer 3: You can, or do we expect that you do?
If it's the latter, then
```suggestion
Use the [troubleshooting topic template](../templates/resources/troubleshooting.md).
```

Reviewer: I suggest:

`For all troubleshooting topics, use the [troubleshooting topic template](../templates/resources/troubleshooting.md).`

### Release Notes

> Announce what's new in Kyma.

Reviewer: ```suggestion
Release notes announce what's new in Kyma.
```

> After an introductory paragraph that outlines the city that's the namesake of the current release, a list briefly presents the new and changed features. Links lead to longer paragraphs that describe the changes in more detail.

Reviewer: I'd be careful with these details. I'm not sure if we're going to follow the city pattern for release naming. Is Kyma 2.0 going to have a city reference? Also, I'd add the link to the release notes template here.

Author: good point. Who could we ask about that? Do we even need an introductory paragraph in release notes?

Reviewer 4: In my opinion, there is no reason to name releases with cities anymore. I created a short poll in the internal slack channel to confirm it.

## Graphics

### Architecture and Flow Diagrams

For information about our diagram style, see [Diagrams](../02-diagrams.md).

### Screenshots

> Wherever possible, present screenshots as simplified user interfaces (SUI). Basically, this means to blur out all UI elements that aren't essential for the task at hand.

Reviewer 3: ```suggestion
Wherever possible, present screenshots as simplified user interfaces (SUI). Basically, this means blurring out all UI elements that aren't essential for the task at hand.
```

For more information, see:

> • <https://www.techsmith.com/blog/simplified-user-interface/>
> • <https://www.tcworld.info/e-magazine/technical-writing/simplified-graphics-and-screenshots-in-software-documentation-1102/>

Reviewer 3: ```suggestion
• https://www.techsmith.com/blog/simplified-user-interface/
• https://www.tcworld.info/e-magazine/technical-writing/simplified-graphics-and-screenshots-in-software-documentation-1102/
```

No need to include these `<>`.

> ## Target Groups

Reviewer 3: ```suggestion
## Target groups
```

You don't use capitalization in other titles. Is there a reason why `Groups` is capitalized here?

Author: [Old habit](https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/47f574d73c8c4cc89368203af2fbafcb.html).

The general assumption is that the audience is familiar with the following terms and does not require the explanation of technical concepts behind them:

- Kubernetes
- Docker and containers

> <!-- Information about generic target groups is based on the SAP Styleguide: https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/546dfb06e80c4005aabc4795d548fe35.html -->

Reviewer: I'd refrain for linking to SAP Styleguide. Also, I'd add the introductory sentence before introducing the target groups. For example:

`These are the target groups identified for Kyma documentation:`

Author: absolutely! This is just for our internal discussion while it's WIP.

### Decision maker

> **Tasks**: Assess the software to make sure that it will meet the company's needs. Needs the facts – not just marketing spin – before signing on the dotted line.

Reviewer 3: ```suggestion
**Tasks**: Assesses the software to make sure that it meets the company's needs. Requires the facts – not just marketing spin – before signing on the dotted line.
```
You have `needs` twice back to back (`... the company's needs. Needs the facts ...`).

> **Goal**: Purchase the right solution for the company, and ensure stakeholders back this decision.

Reviewer: ```suggestion
**Goal**: Choose the right solution for the company, and ensure stakeholders back this decision.
```

Reviewer: You can't really purchase Kyma OS 🤔

### Software developer

> **Tasks**: Interested in technical topics. Solid knowledge of programming languages. Experienced in programming and development projects. Expert in the technical or business area. Uses and contributes to community content.

Reviewer 3: Is "being interested" a task?
Maybe we should change `**Tasks:**` to `**Characteristics:**` or `**Who:**`?

Author: How about "stay up to date with technical topics" ?

Reviewer 3: Well, actually, it was an example, as all of the "tasks" for the software developers are not really tasks, but characteristics. Maybe apart from `Uses and contributes to community content.`.
But we can turn a blind eye to this if it's just me that's bothered by this technicality.

Or, as you suggested, we could rephrase them to make them tasks :)

Reviewer: ... or we can totally get rid of both `Tasks` and `Goal` and just provide descriptions. If we choose to stay with the initial structure, I'd start each "sub-section" with a new line. They display in a single line in GitHub preview:

<img width="1053" alt="Screen Shot 2021-08-10 at 13 24 20" src="https://user-images.githubusercontent.com/35192450/128858775-40c596d5-c9d3-4d73-b2fb-700d13d4c537.png">

Author: good idea, done!

> **Goal**: Develop and enhance software.

Reviewer 3: ```suggestion
**Goal**: Develop and maintain software.
```
`enhance` or `maintain`?
I understand that development is creating something new, and maintenance is taking care of something that already exists (e.g. when there's a bug or another problem with it).

Author: we can just put in all 3 verbs :D

### Admin/Operations

**Tasks**: Deals with installation, upgrades, system troubleshooting.
**Goal**: Support the ongoing operations and evolution of the Kyma implementation.

## Documentation Structure

On the Kyma website, we have five main tabs containing multiple documents each, plus a glossary.

### Overview

> Target Group: Decision Makers (Tech Leads) and newbies.

Reviewer 3: ```suggestion
**Target Group**: Decision Makers (Tech Leads) and newbies.
```

> **Target Group**: Decision Makers (Tech Leads) and newbies.

Reviewer: Here I'd also separate `Target group` from the description and put them in separate lines:

<img width="1042" alt="Screen Shot 2021-08-10 at 13 26 24" src="https://user-images.githubusercontent.com/35192450/128859018-64e2a071-c43c-4bd0-bba1-eef50299aa28.png">

Author: I'm trying to fix the line break with extra lines between the sentences *fingerscrossed

> Contains a quick overview of the idea behind Kyma, a diagram of the main areas with a brief explanation, and, as needed, longer documents going into the details of each main area.

Reviewer: > a diagram of the main areas with a brief explanation

I'd be careful with this one. It might change. I guess I'd mention `basic graphics` in more general way.

Author: Let's update this one once we're closer to release?

### Getting Started

> Target Group: Software Developers who quickly want to see what they can do with Kyma.

Reviewer 3: ```suggestion
**Target Group**: Software Developers who quickly want to see what they can do with Kyma.
```

> Contains a guide/tutorial that covers typical steps you need to get started.

Reviewer 3: ```suggestion
Contains a guide/tutorial that covers typical steps you need to perform to get started.
```
#nitpick

> ### Deep Dive/Tutorials

Reviewer: ```suggestion
### Tutorials
```

> Target Group: Software Developers leveraging all the Kyma functionalities.

Reviewer 3: ```suggestion
**Target Group**: Software Developers leveraging all the Kyma functionalities.
```

> Under this tab, there are subtabs according to main areas (except UI – Busola and CLI are mentioned as needed within the instructions of the respective main area). Documents in the subtabs contain "how-to" instructions that enable users to accomplish a task

Reviewer 3: ```suggestion
Under this tab, there are subtabs according to main areas (except the UI; Busola and CLI are mentioned as needed within the instructions of the respective main area). Documents in the subtabs contain "how-to" instructions that enable users to accomplish a task.
```

Is it Busola in the OS or also Dashboard?

Author: good catch! was Busola when I wrote that, should be Dashboard now. Probably.

> ### Operations

Reviewer: ```suggestion
### Operation guides
```

> Target Group: Admins/Operators who make sure the Kyma cluster is configured as needed and keeps running in a healthy and secure way.

Reviewer 3: ```suggestion
**Target Group**: Admins/Operators who make sure the Kyma cluster is configured as needed and keeps running in a healthy and secure way.
```

> Contains installation and configuration instructions, backup info, troubleshooting guides…

Reviewer: ... also security

Author: good point!

### Technical References

> Target Group: Users who want need to look up specific detailed information.

Reviewer 3: ```suggestion
**Target Group**: Users who want to look up specific detailed information.
```

> Contains the architecture diagrams, configuration charts, etc,; no explanation of concepts or instructions.

Reviewer 3: ```suggestion
Contains the architecture diagrams, configuration charts, etc.; no explanation of concepts or instructions.
```

### Glossary

> Target Group: Anyone who wants to look up terms they’re not familiar with.

Reviewer 3: ```suggestion
**Target Group**: Anyone who wants to look up terms they’re not familiar with.
```

> Explains basic terms, with a focus on terms specific to Kyma.

Reviewer 3: Or 
```suggestion
Explains basic Kyma-related terms.
```
or
```Explains basic Kyma and Kyma-related terms.```
or
```Explains basic Kyma-specific and Kyma-related terms.```

Author: Wasn't sure we are allowed to hyphenate "Kyma", thus the longer phrase.

> ## Content Source

Reviewer: ```suggestion
## Content source
```

> We write the content in [Markdown](https://daringfireball.net/projects/markdown/) and store it in [Git](https://git-scm.com/) repositories.

Reviewer: Not sure we should link to Git here. Maybe to our GitHub repo instead? 🤔 But then, this info would duplicate the one used at the beginning of this doc.

Author: That section was copied from the k1.x content guide. Basically one single main point remains: "we write in Markdown"? :D
